### PR TITLE
PMM-10371, PMM-10473: Telemetry config

### PR DESCRIPTION
**Pull request by ritbl**
Related tickets
- PMM-10371
- PMM-10473

### CODE REVIEW

File: managed/services/telemetry/config.default.yml [hunk 1]
```diff
@@ -245,3 +245,29 @@ telemetry:
...
+      - metric_name: "dbaas_services_count"
+        column: "count"
+
+  - id: DBaaSClustersCount
+    source: PMMDB_SELECT
+    query: select count(distinct cluster) AS count from services where service_name ~ concat('default-', cluster,'-(pxc|rs0)-\d+')
+    summary: "DBaaS Clusters Count"
+    data:
+      - metric_name: "dbaas_clusters_count"
+        column: "count"
```
Comment (maxkondr) [inline managed/services/telemetry/config.default.yml:264]: what is the difference between these 2? as I understand from PMM point of view cluster == service, isn't it?

Comment (BupycHuk) [reply managed/services/telemetry/config.default.yml:264]: No, one cluster might have multiple services, one pod - one service

**Review by BupycHuk — APPROVED** [review]
looks good to me. Have you discussed with Roma about haproxy, cfg and e.t.c. pods?

**Review by artemgavrilov — APPROVED** [review]

**Review by maxkondr — APPROVED** [review]

**Comment by ritbl** [comment]
@BupycHuk Roma is not responding as he is on vacation.

**Pull request merged**